extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java

```diff
@@ -127,8 +149,7 @@ public String getBannerUrl() {
     @Override
     public String getUploaderUrl() throws ParsingException {
         try {
-            return YoutubeChannelExtractor.CHANNEL_URL_BASE +
-                    uploaderInfo.getObject("navigationEndpoint").getObject("browseEndpoint").getString("browseId");
+            return getUrlFromNavigationEndpoint(getUploaderInfo().getObject("navigationEndpoint"));
```

Comment:
Isn't there a way to always get the uploader url in the "/channel/ID" form? This would make it consistent with other urls in the app, preventing bugs such as duplicate subscriptions

Reply:
Yes, there is.

Reply:
@TobiGr: I don't think we should do that, as there are many other ways to get to a /user/name channel in NewPipe. Plus NewPipe could do a simple call to `getId()` or `getUrl()` in `YoutubeChannelExtractor` to get a /channel/id URL, if NewPipe isn't doing that already.